packages/tracing/test/browser/request.test.ts

```diff
@@ -96,15 +96,15 @@ describe('callbacks', () => {
       const noFetchData = { args: fetchHandlerData.args, startTimestamp: fetchHandlerData.startTimestamp };
       const spans = {};
 
-      fetchCallback(noFetchData, alwaysCreateSpan, spans);
+      fetchCallback(noFetchData, always, never, spans);
       expect(spans).toEqual({});
     });
 
     it('does not add fetch request spans if tracing is disabled', () => {
```

Comment:
Similarly, we could combine this test and the `'does not add fetch request headers if tracing is disabled'` test below into a `'neither creates span nor adds headers if tracing is disabled'` test. And again similarly, I think it should be `always`, `always` rather than `always`, `never`, to show that tracing being disabled takes precedence over both of the other options.